src/skin/qml/qmlplayermanagerproxy.cpp

```diff
@@ -35,6 +35,13 @@ QObject* QmlPlayerManagerProxy::getPlayer(const QString& group) {
             [this, group](const QString& trackLocation) {
                 emit loadLocationToPlayer(trackLocation, group);
             });
+    connect(pPlayerProxy,
+            &QmlPlayerProxy::cloneFromGroup,
+            this,
+            [this, group](const QString& sourceGroup) {
+                // FIXME: Calling this slow directly seems hacky
+                m_pPlayerManager->slotCloneDeck(sourceGroup, group);
```

Comment:
typo. Also I'm not a Qt expert, but calling a slot directly is totally legal AFAIK

Reply:
Sure, it works fine. I just looks weird and I'm unsure if that slot is actually intended to be public. Most of the slots in out codebase are public, even those that should be private.